package.json

```diff
@@ -5,12 +5,14 @@
   "private": true,
   "scripts": {
     "test": "env NODE_ENV=test mocha --recursive tests/tests",
+    "cover": "env NODE_ENV=test istanbul cover _mocha -- --recursive tests/tests",
     "start": "node ./app"
   },
   "dependencies": {
     "@onehilltech/blueprint": "^1.1.1",
     "@onehilltech/blueprint-mongodb": "^0.9.1",
     "chai": "^3.5.0",
+    "istanbul": "^0.4.5",
```

Comment:
From my understanding, coveralls will build off of the Istanbul coverage script and integrate with TravisCI. My plan was to get coveralls up and running after TravisCI is integrated